Makefile

```diff
@@ -14,11 +14,14 @@
 
 PACKAGES=$(shell go list ./... | grep -Ev 'vendor|importer|rpc/tester')
 COMMIT_HASH := $(shell git rev-parse --short HEAD)
-BUILD_FLAGS = -tags netgo -ldflags "-X github.com/cosmos/ethermint/version.GitCommit=${COMMIT_HASH}"
+SET_PROJECT_APPNAME = -X main.appName=${APP_NAME} -X main.appNameLong=${APP_NAME} -X app.appName=${APP_NAME} -X app.shortAppName=${APP_NAME} -X mintkey.appName=${APP_NAME} -X types.appName=${APP_NAME} -X types.appNameLong=${APP_NAME} -X evm.appName=${APP_NAME} 
```

Comment:
can you make the names consistent? Having appName, appNameLong, and shortAppName is confusing. You also are just setting everything to APP_NAME, should have another variable for the long app name.